org.scala-ide.sdt.aspects/plugin.xml

```diff
@@ -8,4 +8,5 @@
    <extension-point id="cuprovider" name="Compilation Unit Provider" schema="cuprovider.exsd"/>
    <extension-point id="imagedescriptorselector" name="Image Descriptor Selector" schema="imagedescriptorselector.exsd"/>
    <extension-point id="openactionprovider" name="Open Action Provider" schema="openactionprovider.exsd"/>
+   <extension-point id="lncheck" name="Line Number Check" schema="lncheck.exsd"/>
```

Comment:
Sorry Simon but there is too much mess with merges. I don't want to tinker to not blow up the tree. Below is tree:
*   848942b Merge branch '2.12-compatibility' of https://github.com/scala-ide/sc
|\  
| *   659a57d Merge pull request #1155 from wpopielarski/2.12-compatibility
| |\  
* | | 9786794 Links breakpoint in Breakpoint View with Code Editor for scala cla
| |/  
|/|   
* | 5bba717 Improvements after code review
* | ceab5eb Guards Scala Debug against fetching data outside of presentation com
|/  
*   b7339b4 Merge pull request #1154 from wpopielarski/2.12-compatibility